spec/controllers/api/v1/product_share_controller_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::ProductShareController, type: :controller do
+  describe 'POST #create' do
+    let(:product) { create(:product) }
+    let(:product_share_params) do
+      {
+        product_share: {
+          platform: "email",
+          product_id: product.id
+        },
+        to_email: "example@platan.us",
+        product_link: "platan.us",
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Style/TrailingCommaInHashLiteral: Avoid comma after the last item of a hash.